### PR TITLE
Improve consistency of UI string capitalization

### PR DESCRIPTION
**Pull request by ltrel**
This commit fixes some inconsistencies in the capitalization of various
strings in the UI. For example, 'Page view' now uses a lower case 'v' as
was already the case in 'Continuous view'. This commit also capitalizes
the word 'Roman' in 'Add Roman numeral analysis' as it is a proper noun.

**Complete list of string changes**
In src/appshell/internal/applicationuiactions.cpp:
- Replaced "Enter Full screen" with "Enter full screen" to make the title match the description
- Replaced "Playback Controls" with "Playback controls" to match the capitalization of "Status bar" in the same menu (View>Toolbars)
- Replaced "Note Input" with "Note input" to match the capitalization of "Status bar" in the same menu (View>Toolbars)
- Replaced "Toggle Playback Controls toolbar" with "Toggle 'Playback controls' toolbar" to match the quotation marks and capitalization used in "Toggle 'Status bar'"
- Replaced "Toggle 'Note Input' toolbar" with "Toggle 'Note input' toolbar" to match the capitalization used in "Toggle 'Status bar'"

In src/notation/internal/notationuiactions.cpp:
- Replaced "Page View" with "Page view" to match the capitalization of "Continuous view"
- Replaced "Staff text Properties…" with "Staff text properties…" to match the capitalization of "Staff/part properties…"
- Replaced "Add roman numeral analysis" with "Add Roman numeral analysis" because Roman is a proper noun
- Replaced "Accessibility: get location" with "Accessibility: Get location" to match the capitalization of other accessibility strings such as "Accessibility: Next segment element"
- Replaced "Next Beat (Chord Symbol)" with "Next beat (Chord symbol)" to match its description
- Same as above for "Previous Beat (Chord Symbol)"
- Replaced "Longa Advance (F.B./Chord Symbol)" with "Longa advance (F.B./Chord symbol)" to match sentence case used elsewhere
- Same for all other advance strings
- Replaced "Note duration: Longa" with "Note duration: longa" to match strings such as "Note duration: whole note"

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [N/A] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Tantacrul** [comment]
All of this works for me. Merge away @Eism

**Review by RomanPudashkin — APPROVED** [review]

File: src/notation/internal/notationuiactions.cpp [hunk 1]
```diff
@@ -1111,7 +1111,7 @@ const UiActionList NotationUiActions::m_actions = {
              ),
     UiAction("get-location",
              mu::context::UiCtxNotationOpened,
-             QT_TRANSLATE_NOOP("action", "Accessibility: get location")
+             QT_TRANSLATE_NOOP("action", "Accessibility: Get location")
```
Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:1114]: this seems inconsoostent with...

File: src/notation/internal/notationuiactions.cpp [hunk 2]
```diff
@@ -1569,7 +1569,7 @@ const UiActionList NotationUiActions::m_noteInputActions = {
     UiAction("note-longa",
              mu::context::UiCtxNotationOpened,
              QT_TRANSLATE_NOOP("action", "Longa"),
-             QT_TRANSLATE_NOOP("action", "Note duration: Longa"),
+             QT_TRANSLATE_NOOP("action", "Note duration: longa"),
```
Comment (Jojo-Schmitz) [inline src/notation/internal/notationuiactions.cpp:1572]: ... this

Comment (Jojo-Schmitz) [reply src/notation/internal/notationuiactions.cpp:1572]: Also IMHO "Longa" is a "proper noun"?

Comment (ltrel) [reply src/notation/internal/notationuiactions.cpp:1572]: Wikipedia seems to spell it with a lowercase L when it's in the middle of a sentence https://en.wikipedia.org/wiki/Longa_(music)

Comment (Jojo-Schmitz) [reply src/notation/internal/notationuiactions.cpp:1572]: OK, so probably not a "proper noun", but still the first (and only) word after a colon

File: src/notation/internal/notationuiactions.cpp (hunk 1 above)
Comment (ltrel) [reply src/notation/internal/notationuiactions.cpp:1114]: I've just based it on what other strings beginning with ""Accessibility:" and "Note duration:" were doing, but yes there is a wider inconsistency here. Is there a preferred style that could be used for both groups?

File: src/notation/internal/notationuiactions.cpp (hunk 2 above)
Comment (ltrel) [reply src/notation/internal/notationuiactions.cpp:1572]: I've done some digging and checked which of the groups with colons in the middle use an uppercase letter following the comma as of after this PR. Shall I make another PR to change the ones that aren't using an uppercase letter? (given that these ones seem to be in the minority)

Using uppercase:
- "Accessibility:"
- "Note input:"
- "Screen:"
- "Page:"
- "Toggle accidental:"

Not using uppercase
- "Grace:"
- "Note duration:"

- "Note input: rest" (I missed an individual string here, this goes against the rest of the group)

**Pull request merged**